### PR TITLE
Update rfdetr notebooks

### PR DESCRIPTION
**Pull request by yutong-xiang-97**
## What has changed and why?

Use distinct datasets for pretraining and fine-tuning in the RFDETR tutorial notebook. 
Make pretraining and fine-tuning into individual sections.

## How has it been tested?

Run on Colab

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [ ] Yes
- [x] Not needed (internal change)

## Did you update the documentation?

- [ ] Yes
- [x] Not needed (internal change without effects for user)

### CODE REVIEW

File: examples/notebooks/rfdetr.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this notebook we will demonstrate how you can use [LightlyTrain](https://docs.lightly.ai/train/stable/index.html) to pretrain an [RF-DETR model](https://github.com/roboflow/rf-detr). To this end, we will first use the raw images (**no labels**) from the [Coconuts dataset](https://universe.roboflow.com/traindataset/coconuts-plj8h) for pretraining and then fine-tuning on the labeled dataset.\n",
+    "In this notebook we will demonstrate how you can use [LightlyTrain](https://docs.lightly.ai/train/stable/index.html) to pretrain an [RF-DETR model](https://github.com/roboflow/rf-detr) from Roboflow. For now, `rfdetr` only supports training with datasets in COCO JSON format. To this end, for pretraining we use the raw images (**no labels**) from [the COCO-minibatch dataset](https://github.com/giddyyupp/coco-minitrain), a subset of the COCO dataset with 25k images, and for fine-tuning we use the Roboflow's [Coconut Custom Dataset](https://universe.roboflow.com/ravi-mgvlz/coconut-custom-dataset).\n",
```
Comment (liopeer) [inline examples/notebooks/rfdetr.ipynb]:
```suggestion
    "In this notebook we will demonstrate how you can use [LightlyTrain](https://docs.lightly.ai/train/stable/index.html) to pretrain an [RF-DETR model](https://github.com/roboflow/rf-detr) from Roboflow. For now, `rfdetr` only supports training with datasets in COCO JSON format. To this end, for pretraining we use the raw images (**no labels**) from [the COCO-minitrain dataset](https://github.com/giddyyupp/coco-minitrain), a subset of the COCO dataset with 25k images, and for fine-tuning we use the Roboflow's [Coconut Custom Dataset](https://universe.roboflow.com/ravi-mgvlz/coconut-custom-dataset).\n",
```

File: examples/notebooks/rfdetr.ipynb [hunk 2]
```diff
@@ -54,9 +54,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Download Dataset\n",
+    "## Pretrain on COCO-minitrain-25k Dataset\n",
     "\n",
-    "For now, `rfdetr` only supports training with datasets in COCO JSON format. They can be directly downloaded via their API:\n"
+    "We use [the COCO-minibatch dataset](https://github.com/giddyyupp/coco-minitrain), a subset of the COCO dataset with 25k images for pretraining the RF-DETR model. This dataset is available in COCO JSON format.\n",
```
Comment (liopeer) [inline examples/notebooks/rfdetr.ipynb]:
```suggestion
    "We use [the COCO-minitrain dataset](https://github.com/giddyyupp/coco-minitrain), a subset of the COCO dataset with 25k images for pretraining the RF-DETR model. \n",
```
I would leave the info on the COCO JSON format away, since we don't actually use the labels for COCO-minitrain.

File: examples/notebooks/ultralytics_yolo.ipynb [hunk 3]
```diff
@@ -84,7 +84,7 @@
    "id": "7",
    "metadata": {},
    "source": [
-    "... and since Lightly**Train** does not require any labels, we can can also confidently delete all the labels:"
+    "... and since Lightly**Train** does not require any labels, we can also confidently delete all the labels:"
```
Comment (liopeer) [inline examples/notebooks/ultralytics_yolo.ipynb:87]: Thx!

File: examples/notebooks/rfdetr.ipynb [hunk 4]
```diff
@@ -150,9 +141,44 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fine-tune an RF-DETR Model\n",
     "\n",
-    "You can use `rfdetr` for fine-tuning directly."
+    "You can use the `rfdetr` package for fine-tuning directly."
```
Comment (liopeer) [inline examples/notebooks/rfdetr.ipynb]: Nit: 
```suggestion
    "You can directly use the `rfdetr` package for fine-tuning."
```

**Review by liopeer — APPROVED** [review]
